### DISCUSSION TITLE
Bitwarden Send? Is this available or will it be soon?

### DISCUSSION THREAD

**Discussion by ahsanshah** [dani-garcia/vaultwarden#1547] [Q&A]
Hi.  Is there any plans to enable the Bitwarden Send feature in this image? The image shows 1.19 and the Send official release notes need 1.40?  Can anyone advise how the releases/features like this in the official BW release get incorporates to the Rust image?

**Comment by BlackDex** [comment]
You can use the `testing` image if you want. It's not in `latest`
